src/material-examples/text-field-autosize-basic/text-field-autosize-basic-example.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <label>Autosize textarea basic example</label>
+  <textarea cdkTextareaAutosize></textarea>
+</div>
+<div>
+  <label>Autosize textarea with upper and lower bounds set</label>
```

Comment:
The `label` should be associated with the `textarea`, wither by using the `for` attribute on the label or by putting the `textarea` inside the `label`.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
Added `for` - thanks